src/components/DynamicForm/components/DynamicFormContainer.jsx

```diff
@@ -171,13 +171,26 @@ class DynamicFormContainer extends React.Component {
       </React.Fragment>
     );
   };
-  
+
+  renderDisabledSubmitBtn = () => {
+    return (
+      <React.Fragment>
+        <hr className="form-hline" />
+        <input
+          className="form-btn--disabled"
+          disbaled
+          value="Can't Submit Yet"
```

Comment:
what do you think of changing this to "Incomplete"? It's visually closer in size to Submit

Reply:
i can go both ways. But I chose "cant submit yet" because it sounded friendlier, and still lets the user know their form is missing something. but if thats not a good enough reason,  ill change it !